config/software/ruby.rb

```diff
@@ -41,6 +41,7 @@
 # and that's the only one we will ever use.
 dependency "libiconv"
 
+version("2.4.6")      { source sha256: "254f1c1a79e4cc814d1e7320bc5bdd995dc57e08727d30a767664619a9c8ae5a" }
```

Comment:
This might not be the right hash,  wasn't actually sure how to pull this but if anyone knows off hand happy to go grab it.